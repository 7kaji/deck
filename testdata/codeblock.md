# Code Block

```go
package main

import "fmt"

func main() {
	fmt.Println("Hello, 世界")
}
```

```ts
const anExampleVariable = "Hello World"
console.log(anExampleVariable)
```

```mermaid
graph TD
    A[md.Contents] --> B[deck.Slide];
    B --> C[Google Slides];
```

```
Hello World
```

<!-- {"layout":"title-and-body"} -->
