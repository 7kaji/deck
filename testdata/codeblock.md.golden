[
  {
    "layout": "",
    "titles": [
      "Code Block"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Blocks"
    ],
    "code_blocks": [
      {
        "language": "go",
        "value": "package main\n\nimport \"fmt\"\n\nfunc main() {\n\tfmt.Println(\"Hello, 世界\")\n}\n"
      },
      {
        "language": "ts",
        "value": "const anExampleVariable = \"Hello World\"\nconsole.log(anExampleVariable)\n"
      },
      {
        "language": "mermaid",
        "value": "graph TD\n    A[md.Contents] --\u003e B[deck.Slide];\n    B --\u003e C[Google Slides];\n"
      },
      {
        "value": "Hello World\n"
      }
    ]
  }
]